### DISCUSSION TITLE
XMSBT patches that include multiple regions falls through to the last region alphabetically

### DISCUSSION THREAD

**Discussion by ThatNintendoNerd** [Raytwo/ARCropolis#439] [Issues] [answered]
### User Setup

System Firmware Version: **14.1.2**
Atmosphère Version: **1.4.0** (sysMMC)
Smash Version: **13.0.1**
ARCropolis Version: **3.8.0**

### Bug Info

If a mod includes XMSBT patches for multiple regions, the last region alphabetically will be used, regardless of the console's region or game language setting. For example, if my Switch system is configured to the Americas and my game English (`us_en`), but there is a patch for any of these regions:
- `us_es`
- `us_fr`
- `zh_cn`
- `zh_tw`

The patch last in the list will be used in-game. In this example, Traditional Chinese (`zh_tw`) is what gets used if it exists. Despite Traditional Chinese being the last region ID, I have confirmed that IDs are independent of the issue. Removing the two Chinese patches results in Western French (`us_fr`) to be used and not the Korean patch (`kr_ko`) which internally comes before the two Chinese regions.

### How to Reproduce

1. Install the [Dive to the Heart Configurator](https://gamebanana.com/mods/393112) UI package only. Alternatively—for no DLC—[25 m](https://gamebanana.com/mods/309758).
2. Launch the game and hover over either of the stage mods on the Stage Select screen.
3. Under the current circumstances, Hollow Bastion / Dive to the Heart should appear with its Traditional Chinese name of "深潛心界" and 25 m as "25m" (no space for Asian languages, spaces between the number and unit of measurement everywhere else).

### Other Notes

This has been an ongoing issue, I just have not gotten around to submitting an issue for it.

**Comment by Raytwo** [comment, marked as answer]
It be fixed
